### PR TITLE
Beta doc updates

### PR DESCRIPTION
**Pull request by nrichers**
## Description

This PR includes several documentation tweaks for the beta:

- Add a standard page header, as used by other books. (To remove this header in the future, simply delete `page_header.html`.)
- Replace the beta note on the overview page with the standard Elastic `beta[]` notice. 
- Simplify some of the revised wording on the overview page

Here's some output details:

image

image

## Related Issues

Relates to https://github.com/elastic/ecctl/pull/22

## Motivation and Context

These are consistency changes meant to follow the standard for Elastic books. 

## How Has This Been Tested?

Test built locally with `docbldecctl` and proofed the output.

## Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (improves code quality but has no user-facing effect)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation

## Readiness Checklist

- [ ] My code follows the code style of this project
- [ ] My change requires a change to the documentation
- [x] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed

### CODE REVIEW

File: docs/ecctl-getting-started.asciidoc [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
 Benefits of {p}:
 
-* Easier to use for ops than using the user console using the RESTful API directly
+* Easier to use than the Cloud UI or using the RESTful API directly
 * Helps you automate the deployment lifecycle
 * Provides a foundation for integration with other tools, such as Terraform and Ansible
 
-Note that, right now, {p} targets ECE installations only. There are, however, plans
-to support public Elasticsearch Service API in the future.
+Right now, {p} supports ECE installations only, but we plan to support a public Elasticsearch Service API in the future.
```
Comment (nrichers) [inline docs/ecctl-getting-started.asciidoc]: I left the bit about the future public ESS API in there, but we typically don't discuss future functionality in the docs. Could we perhaps just say `Right now, {p} supports ECE installations only.`?

Comment (ppapapetrou76) [reply docs/ecctl-getting-started.asciidoc]: Yeah it was me who asked Ana to add it but you are probably right @nrichers . Let's stick on the way we are doing things already

File: docs/ecctl-getting-started.asciidoc [hunk 2]
```diff
@@ -1,20 +1,19 @@
...
+beta[]
 
 {p} is the CLI on steroids for the Elastic Cloud Enterprise API. It wraps
 typical operations commonly needed by ECE operators within a
 single command line tool.
 
 Benefits of {p}:
 
-* Easier to use for ops than using the user console using the RESTful API directly
+* Easier to use than the Cloud UI or using the RESTful API directly
```
Comment (ppapapetrou76) [inline docs/ecctl-getting-started.asciidoc]: @nrichers I assume we are using the term Cloud UI across the docs, right? we had a chat with @zanbel yesterday which one to use :) just double-checking

Comment (nrichers) [reply docs/ecctl-getting-started.asciidoc]: @ppapapetrou76 this is how we define references to the user console for ECE: `:console:       Cloud UI`. So "Cloud UI" is what we use in the ECE docs officially and always have. This is different from SaaS--at some point we probably want to consolidate names, but for now I would suggest using the ECE one.

File: docs/ecctl-getting-started.asciidoc (hunk 1 above)
Comment (nrichers) [reply docs/ecctl-getting-started.asciidoc]: Roger, I'll push a commit to remove the reference to future functionality. As soon as we support the public ESS API, we can update the ecctl docs again.

**Pull request merged**